app/src/main/java/nyc/c4q/yuliyakaleda/meme_ifyme/MainActivity.java

```diff
@@ -24,48 +32,21 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
-        take = (Button) findViewById(R.id.take);
-        choose = (Button) findViewById(R.id.choose);
-
+        initializeViews();
         setEventListener(true);
-
     }
-    @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-
-        Bitmap imageBitmap = null;
-        switch (requestCode) {
-            case SELECT_PICTURE:
-                if (resultCode == RESULT_OK) {
-                    imageUri = data.getData();
-                    try {
-                        imageBitmap = MediaStore.Images.Media.getBitmap(this.getContentResolver(), imageUri);
-                        // http://stackoverflow.com/a/10281667, image is too big so you gotta resize
-                        imageBitmap = Bitmap.createScaledBitmap(imageBitmap, 200, 200, true);
-                    } catch (IOException e) {
-                    }
-                }
-                break;
 
-            case REQUEST_IMAGE_CAPTURE:
-                if (resultCode == RESULT_OK) {
-                    Bundle extras = data.getExtras();
-                    imageBitmap = (Bitmap) extras.get("data");
-                }
-                break;
-        }
-        Intent sentTo = new Intent(MainActivity.this, SecondActivity.class);
-        sentTo.putExtra("bitmap", imageBitmap);
-        startActivity(sentTo);
+    public void initializeViews() {
+        take = (ImageButton) findViewById(R.id.take);
+        choose = (ImageButton) findViewById(R.id.choose);
     }
```

Comment:
instead of having a two-line code, there is a 4-line code now...maybe it was not really a great idea to isolate two lines of code in a separate methos